bin/setup

```diff
@@ -1,49 +1,60 @@
 #!/usr/bin/env ruby
-require 'pathname'
+# frozen_string_literal: true
+
 require 'fileutils'
 
 # path to your application root.
-APP_ROOT = Pathname.new File.expand_path('..', __dir__)
+APP_ROOT = File.expand_path('..', __dir__)
+
+def system!(*args)
+  system(*args, exception: true)
+end
+
+FileUtils.chdir APP_ROOT do
+  # This script is a way to set up or update your development environment automatically.
+  # This script is idempotent, so that you can run it at any time and get an expectable outcome.
+  # Add necessary setup steps to this file.
 
-Dir.chdir APP_ROOT do
   puts '== Installing dependencies =='
-  system 'gem install bundler --conservative'
-  system 'bundle check || bundle install --without seven_zip'
+  system! 'gem install bundler --conservative'
+  system('bundle check') || system!('bundle install --without seven_zip')
```

Comment:
Looks like we've removed `seven_zip_ruby` from the gemfile.  We can probably remove it here as well:
```suggestion
  system('bundle check') || system!('bundle install')
```